api/working-with-extensions/publishing-extension.md

```diff
@@ -38,7 +38,7 @@ $ cd myExtension
 $ vsce package
 # myExtension.vsix generated
 $ vsce publish
-# <publisherID>.myExtension published to VS Code Marketplace
+# <publisher name>.myExtension published to VS Code Marketplace
```

Comment:
It is the publisherID not the publisher name in the unique extension ID

Reply:
This is actually the only place in the whole document where we're talking about the publisher ID. That's why I tried to stick to the 'old' naming scheme and describe the current mapping in lines 101-104